src/mono/mono/eglib/glib.h

```diff
@@ -1371,12 +1134,7 @@ gchar    *g_convert            (const gchar *str, gssize len,
 extern const guchar g_utf8_jump_table[256];
 
 gboolean  g_utf8_validate      (const gchar *str, gssize max_len, const gchar **end);
-gunichar  g_utf8_get_char_validated (const gchar *str, gssize max_len);
-#define   g_utf8_next_char(p)  ((p) + g_utf8_jump_table[(guchar)(*p)])
```

Comment:
Why is this being removed?

In particular: `g_utf8_jump_table` is not being removed, so there's not even any savings.